implementation.js

```diff
@@ -14,27 +14,41 @@ var $toString = callBound('Object.prototype.toString');
 var stackDesc = gOPD($Error.prototype, 'stack');
 var stackGetter = stackDesc && stackDesc.get && callBind(stackDesc.get);
 
+var domExceptionClonable = !!(
+	$structuredClone
+	&& typeof DOMException === 'function'
+	&& $structuredClone(new DOMException()) instanceof $Error
+);
+
 module.exports = function isError(arg) {
 	if (!arg || (typeof arg !== 'object' && typeof arg !== 'function')) {
 		return false; // step 1
 	}
 
+	var str = $toString(arg);
+
+	if (str === '[object DOMException]') {
+		return true;
+	}
```

Comment:
this check can't be relied upon in an environment with Symbol.toStringTag (if the arg has Symbol.toStringTag in it) - otherwise `{ [Symbol.toStringTag]: 'DOMException' }` will return true, and it must not.

Reply:
Yeah, as noted in https://github.com/es-shims/Error.isError/issues/2 (OP), you have 2 options for dealing with `DOMException`s, neither of which are 100% foolproof.

Of the two (spoofed `toStringTag` vs cross-realm `DOMException`s), checking `toStringTag` is probably less likely to give wrong results in practice, as objects with `toStringTag` set to the specific value of `"DOMException"` are unlikely (except for polyfills).

Reply:
I generally would prefer to have false negatives than false positives in things like this.